qemu/tests/virtio_fs_share_data.py

```diff
@@ -82,6 +88,36 @@ def get_viofs_exe(session):
     fio_options = params.get('fio_options')
     io_timeout = params.get_numeric('io_timeout')
 
+    # xfstest config
+    cmd_xfstest = params.get('cmd_xfstest')
+    xfstest_pkg = params.get('xfstest_pkg')
+    cmd_unpack_xfstest = params.get('cmd_unpack_xfstest')
+    cmd_yum_libacl = params.get('cmd_yum_libacl')
+    cmd_rm_git = params.get('cmd_rm_git')
+    cmd_make_xfs = params.get('cmd_make_xfs')
+    cmd_setenv = params.get('cmd_setenv')
+    cmd_check_test = params.get('cmd_check_test')
+    blacklist = params.get('blacklist')
+
+    # xfstest-nfs config
+    cmd_export = params.get('cmd_export')
+    cmd_xfstest_nfs = params.get('cmd_xfstest_nfs')
+    cmd_umount = params.get('cmd_umount')
```

Comment:
Please use method of class `Nfs` to implement this part from `./avocado-vt/virttest/nfs.py`

Reply:
Corrected